package/piksi_system_daemon/src/ports.c

```diff
@@ -202,36 +202,40 @@ static int port_configure(port_config_t *port_config)
     port_config->opts_get(opts, sizeof(opts), &port_config->opts_data);
   }
 
-  char cmd[512];
-  snprintf(cmd, sizeof(cmd),
-           "zmq_adapter %s %s %s",
+  char cmd_options[512];
+  snprintf(cmd_options, sizeof(cmd_options),
```

Comment:
The ->opts member seems like it can grow to be fairly lengthy, is it worth checking that snprintf didn't truncate?

Reply:
nah. The maximum length in the table is 32 for ->opts.